pkg/install/generate.go

```diff
@@ -284,6 +291,9 @@ func GenerateInstallerJob(
 			ActiveDeadlineSeconds: &deadline,
 			BackoffLimit:          &backoffLimit,
 			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: installerPodLabels,
+				},
```

Comment:
Lets apply use the install job label to the pod template as well, so we know it's both a pod for this cluster deployment, *and* it's an install job. Otherwise it will probably get confused when we get to uninstalls, or possibly other things as well.

Reply:
not sure If I understand this correctly, Do we want the install job label on installerPodLabels? It has the InstallJobLabel: "true", is that not enough?

Reply:
My mistake, you're right, it's already there!

Lets drop the namespace label though, we know that from the pod.